lte/gateway/c/core/oai/include/s1ap_types.h

```diff
@@ -84,10 +84,12 @@ typedef struct s1ap_handover_state_s {
   mme_ue_s1ap_id_t mme_ue_s1ap_id;
   uint32_t source_enb_id;
   uint32_t target_enb_id;
-  enb_ue_s1ap_id_t
-      target_enb_ue_s1ap_id : 24;  ///< Unique UE id over eNB (24 bits wide)
-  sctp_stream_id_t target_sctp_stream_recv;  ///< eNB -> MME stream
-  sctp_stream_id_t target_sctp_stream_send;  ///< MME -> eNB stream
+  enb_ue_s1ap_id_t source_enb_ue_s1ap_id : 24;
+  enb_ue_s1ap_id_t target_enb_ue_s1ap_id : 24;
+  sctp_stream_id_t source_sctp_stream_recv;  ///< source eNB -> MME stream
+  sctp_stream_id_t target_sctp_stream_recv;  ///< target eNB -> MME stream
+  sctp_stream_id_t source_sctp_stream_send;  ///< MME -> source eNB stream
+  sctp_stream_id_t target_sctp_stream_send;  ///< MME -> target eNB stream
```

Comment:
you need to update protos and state conversions.

Reply:
interestingly the code notes
`  // Handover status. We intentionally do not persist all of this state since
  // it's time sensitive; if the MME restarts during a HO procedure the RAN
  // will abort the procedure due to timeouts, rendering this state useless.
  s1ap_handover_state_t s1ap_handover_state;`
 But still we store it?

Reply:
We only store a subset of handover related state; the source target info is ephemeral though and doesn't need to be stored. Honestly, we can probably skip all of it though.

Reply:
Yes we discussed about this, shall we then remove all the state?

Reply:
I think we should remove it. Lets do it in a separate PR